week-2/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  if (number < 0) {
+    return "negative";
+  } else if (number >= 0) {
```

Comment:
You don't need to check `number >= 0` here since you know that condition must already be satisfied (because the number is not less than zero if the code gets as far as the `else`).